servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -921,23 +921,35 @@ void reflection_process(uint ref_index, vec3 vertex, hvec3 ref_vec, hvec3 normal
 
 		vec3 local_ref_vec = (reflections.data[ref_index].local_matrix * vec4(ref_vec, 0.0)).xyz;
 
+		float mip = sqrt(roughness) * MAX_ROUGHNESS_LOD;
+		float mip_min = pow(roughness, 2.0) * MAX_ROUGHNESS_LOD; // Ensures fully rough materials don't have reflection contact hardening.
```

Comment:
Same as above, I don't trust mobile compilers to realize that the pow can be computed with a single multiplication.